provider/datagen/src/source.rs

```diff
@@ -129,13 +129,19 @@ impl SourceData {
         tag: &str,
         _use_default_here: crate::CldrLocaleSubset,
     ) -> Result<Self, DataError> {
-        Ok(Self {
+        Ok(self.with_cldr_for_tag_2(tag))
```

Comment:
Please don't. It adds complexity to this file just so you don't have to write `Default::default` in one (?) location?

Reply:
hm? I added these internal infallible functions so I don't need to call `unwrap()`.

Reply:
ah fallibility

Reply:
It's only used in `latest_tested` which already uses `unwrap()` (well, the code you moved from `Default`). This PR already does a lot of things and this is really tangential